esterni/analisi-dei-requisiti/components/casi-duso/app/registrazione.tex

```diff
@@ -206,18 +244,47 @@ \subsubsection{UUC1.3.7: Conferma password registrazione non valida}%
         \end{enumerate}
 \end{description}
 
-\subsubsection{UUC1.3.8: Dati anagrafici registrazione non validi}%
-\label{subs:UUC1.3.8}
+\subsubsection{UUC1.3.10: Nome registrazione non valido}%
+\label{subs:UUC1.3.10}
 \begin{description}
-  \item[Caso d'uso:] UUC1.3.8;
-  \item[Titolo:] Dati anagrafici registrazione non validi;
+  \item[Caso d'uso:] UUC1.3.10;
+  \item[Titolo:] Nome registrazione non valido;
+  \item[Attori primari:] utente non autenticato;
+  \item[Precondizione:] l'utente non inserisce il proprio nome;
+  \item[Postcondizione:] l'applicazione mobile comunica all'utente l'errore;
+  \item[Scenario Principale:]
+        \begin{enumerate}
+          \item l'utente visualizza il messaggio d'errore, in quanto deve essere riempito il campo con il proprio nome.
+        \end{enumerate}
+\end{description}
+
+\subsubsection{UUC1.3.11: Cognome registrazione non valido}%
+\label{subs:UUC1.3.11}
+\begin{description}
+  \item[Caso d'uso:] UUC1.3.11;
+  \item[Titolo:] Cognome registrazione non valido;
+  \item[Attori primari:] utente non autenticato;
+  \item[Precondizione:] l'utente non inserisce il proprio cognome;
+  \item[Postcondizione:] l'applicazione mobile comunica all'utente l'errore;
+  \item[Scenario Principale:]
+        \begin{enumerate}
+          \item l'utente visualizza il messaggio d'errore, in quanto deve essere riempito il campo con il proprio cognome.
+        \end{enumerate}
+\end{description}
+
+\subsubsection{UUC1.3.12: Data di nascita registrazione non valida}%
+\label{subs:UUC1.3.12}
+\begin{description}
+  \item[Caso d'uso:] UUC1.3.12;
+  \item[Titolo:] Data di nascita registrazione non valida;
   \item[Attori primari:] utente non autenticato;
-  \item[Precondizione:] l'utente non inserisce tutti i campi che riguardano i dati anagrafici;
+  \item[Precondizione:] l'utente non inserisce la propria data di nascita;
   \item[Postcondizione:] l'applicazione mobile comunica all'utente l'errore;
   \item[Scenario Principale:]
         \begin{enumerate}
-          \item l'utente visualizza il messaggio d'errore, in quanto devono essere compilati tutti i campi dei dati anagrafici.
+          \item l'utente visualizza il messaggio d'errore, in quanto la data di nascita inserita deve rispettare il formato \textit{yyyy-mm-dd}.
```

Comment:
questo non è corretto; la data inserita dall'utente non ha necessariamente quel formato. il criterio di approvazione della data di nascita credo debba essere solo che il tizio è maggiorenne.

riformulerei in "l'utente ha più di 18 anni e la data è una data valida" dove con valida si intende ad esempio NON 30 febbraio

(da correggere anche in tutti gli altri punti dove c'è la data di nascita)